PHC-Android/PHC/src/main/res/values/strings.xml

```diff
@@ -152,16 +152,24 @@
     <string name="enter_code_prompt">Enter a code here: </string>
     <string name="scan_code_prompt">Or, scan a QR code: </string>
     <string name="input_confirmation_text">The code you entered was: </string>
-    <string name="scan_confirmation_text">The scan result was: </string>
-
-    <!-- Exit Activity -->
-    <!-- TODO: rename exit to checkout -->
-    <string name="exit_qr_scanner">Scan QR Code and Submit</string>
-    <string name="exit_experience_title">Did you have a positive experience?</string>
-    <string name="exit_services_title"> Did you receive the services you were looking for? </string>
-    <string name="exit_comment_title">Comments</string>
-    <string name="exit_yes">Yes</string>
-    <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="scan_confirmation_texts">The scan result was: </string>
+
+    <!-- Checkout Activity -->
+
+    <string name="checkout_qr_scanner">Scan QR Code and Submit</string>
+    <string name="checkout_radio_group"></string>
+    <string name="checkout_experience_title">Rate your experience</string>
+    <string name="checkout_services_title"> Of these services, what are your top 2 priorities?</string>
+    <string name="checkout_comment_title">Other comments? </string>
+    <string name="checkout_hello_blank_fragment">Hello blank fragment</string>
```

Comment:
they're all used for the checkout page. is there a specific one that doesn't make much sense?